### PR TITLE
Added audit logging section on landing page

### PR DESCRIPTION
**Pull request by bandinib-amzn**
*Issue #, if available:*

*Description of changes:*
Added audit logging section on landing page

**Screenshots:**
Mockup:
image

Implementation:
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/configuration/panels/get-started.tsx [hunk 1]
```diff
@@ -166,6 +166,26 @@ export function GetStarted(props: AppDependencies) {
       <EuiSpacer size="l" />
 
       <EuiSteps steps={setOfSteps} />
+
+      <EuiText size="s" color="subdued" grow={false}>
+        <h3>Optional: Configure audit logs</h3>
+        <p>
+          Elasticsearch audit logs are records that provide documentary evidance of security
```
Comment (tianleh) [inline public/apps/configuration/panels/get-started.tsx:173]: Minor: evidance -> evidence

Comment (bandinib-amzn) [reply public/apps/configuration/panels/get-started.tsx]: Corrected.

File: public/apps/configuration/panels/get-started.tsx [hunk 2]
```diff
@@ -166,6 +166,26 @@ export function GetStarted(props: AppDependencies) {
       <EuiSpacer size="l" />
 
       <EuiSteps steps={setOfSteps} />
+
+      <EuiText size="s" color="subdued" grow={false}>
+        <h3>Optional: Configure audit logs</h3>
```
Comment (yuxitang-amzn) [inline public/apps/configuration/panels/get-started.tsx:171]: Please use EuiTitle

Comment (bandinib-amzn) [reply public/apps/configuration/panels/get-started.tsx:171]: Changed

**Pull request merged**